pkgs/top-level/all-packages.nix

```diff
@@ -14603,6 +14603,8 @@ in
 
   pybind11 = pythonPackages.pybind11;
 
+  pylode = python3Packages.callPackage ../misc/pylode { };
```

Comment:
Although there's a counter example here right above, there's a sort of agreement to not use a `callPackage` that's out of scope. Meaning that in this case I'd suggest to use regular `callPackage` here and inside the file to use:

```nix
python3.pkgs.buildPythonApplication {
  # ...
}
```